server/aoprotocol.py

```diff
@@ -230,14 +222,16 @@ def net_cmd_ch(self, _):
         self.ping_timeout = asyncio.get_event_loop().call_later(self.server.config['timeout'], self.client.disconnect)
 
     def net_cmd_askchaa(self, _):
-        """ Ask for the counts of characters/evidence/music
+        """ Ask for the counts of characters/evidence/music, along with setting active_player on the client.
 
         askchaa#%
 
         """
         char_cnt = len(self.server.char_list)
         evi_cnt = 0
         music_cnt = sum([len(x) for x in self.server.music_pages_ao1])
+        self.client.active_player = True
+        self.client.send_command('ID', self.client.id, self.server.software, self.server.get_version_string())
```

Comment:
this is already sent when the client sends HI, why send it again?

Reply:
Because the hi handshake is sent when the client first clicks on the name of the server before actually connecting. We want to send this so that we can properly trigger the client-is-playing log.